### ISSUE TITLE
What's the weight decay of pretrain

### ISSUE THREAD

**Issue by luyvlei** [Tencent/HunyuanVideo#197]
Thanks open source for such good work. We are using this model on downstream tasks and would like to ask what the weight_decay is when the model pretrain?
